benchmark.py

```diff
@@ -559,15 +577,22 @@ def main(argv):
   bazel_bench_uid = datetime.datetime.utcnow().strftime("%Y%m%d%H%M%S")
 
   bazel_bin_base_path = FLAGS.bazel_bin_dir or BAZEL_BINARY_BASE_PATH
+  bazel_bin_identifiers = []
 
   for bazel_commit in bazel_commits:
+    bazel_bin_path = _build_bazel_binary(bazel_commit, bazel_clone_repo,
+                                            bazel_bin_base_path)
```

Comment:
does the indentation look weird only here on github?

Reply:
Fixed.